ts-libs/linea-native-libs/src/scripts/build.ts

```diff
@@ -38,15 +38,15 @@ async function getReleaseAssetUrl(authToken: string, nativeLibReleaseTag: string
   }
 
   const asset = release.assets.find((asset: any) => asset.name.includes(nativeLibReleaseTag));
-  return `https://${authToken}:@api.github.com/repos/Consensys/zkevm-monorepo/releases/assets/${asset.id}`;
+  return `https://api.github.com/repos/Consensys/linea-monorepo/releases/assets/${asset.id}`;
```

Comment:
Does this work? My understanding is that we need GITHUB token to access https://api.github.com/... releases/assets even if the repo is open source.

Reply:
yes it works without the github token :)